packages/preferences-persistence/src/migrations/legacy-local-storage-data/test/index.js

```diff
@@ -169,7 +169,6 @@ describe( 'convertLegacyData', () => {
 		  "core/edit-site": {
 		    "complementaryArea": "edit-site/global-styles",
 		    "fixedToolbar": true,
-		    "focusMode": true,
```

Comment:
This is causing unit test failure; probably need to update the assertion or snapshot.